integration/v3_grpc_test.go

```diff
@@ -1898,7 +1898,7 @@ func TestV3LargeRequests(t *testing.T) {
 		expectError     error
 	}{
 		// don't set to 0. use 0 as the default.
-		{1, 1024, rpctypes.ErrGRPCRequestTooLarge},
+		{256, 1024, rpctypes.ErrGRPCRequestTooLarge},
```

Comment:
can we explain why we need this change?

Reply:
explained above.
https://github.com/etcd-io/etcd/pull/11427#discussion_r355620014